databricks-sdk-java/src/main/java/com/databricks/sdk/mixin/JobsExt.java

```diff
@@ -0,0 +1,51 @@
+package com.databricks.sdk.mixin;
+
+import com.databricks.sdk.core.ApiClient;
+import com.databricks.sdk.service.jobs.*;
+import java.util.Collection;
+
+public class JobsExt extends JobsAPI {
+
+  public JobsExt(ApiClient apiClient) {
+    super(apiClient);
+  }
+
+  public JobsExt(JobsService mock) {
+    super(mock);
+  }
+
+  @Override
+  public Run getRun(GetRunRequest request) {
```

Comment:
Ditto here, we may want to have a separate doccomment.

Reply:
> Ditto here

I may have missed another comment of yours.
I added a docstring to this method in this commit: [e9eec92](https://github.com/databricks/databricks-sdk-java/pull/324/commits/e9eec92a12a25950a5a71478f538e16ee38043b0) Is this what you were looking for?

Reply:
Ah, you were referring to this? https://github.com/databricks/databricks-sdk-py/pull/725#discussion_r1718291278